opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -1,622 +1,179 @@
-#
-# This properties file allows you to set global properties for the opennms
-# application.  These properties will be set as system properties at the
-# beginning of Java startup.  Any properties set on the Java command-line
-# with -Dproperty=value will be overridden by these values.
-#
-
-# ###### ICMP ######
-# OpenNMS provides three ICMP implementations. JICMP (legacy, IPv4-only),
-# JNA (supports both IPv4 and IPv6), and JICMP6.
-# 
-# The JICMP implementation is what has traditionally been used in OpenNMS
-# since 1.0, uses JNI.  It requires you to install a separate package
-# (JICMP) which contains a shared library for interfacing with your system's
-# ICMP APIs.
-# 
-# The JICMP6 core library is a version of the JICMP codebase which can speak
-# ICMPv6, instead of ICMPv4. The OpeNMS JICMP6 pinger supports both ICMPv4
-# and ICMPv6.  It delegates all ICMPv4 ping requests to the original JICMP
-# JniPinger above and uses the JICMP6 library for ICMPv6 packets.  This is
-# the default for OpenNMS 1.9.90 and up.
-#
-# Finally, the JNA implementation is written from the ground up to support
-# IPv4 and IPv6, and takes advantage of the JNA project's ability to access
-# native APIs without needing to distribute separate shared libraries.  It
-# is, however, not as performant as the JICMP6 pinger, so it is not
-# recommended unless you are in an environment which requires it.  This is
-# the default ICMP implementation used in the remote poller, since it does
-# not rely on any external native code to be installed outside of the JVM.
-# 
-# To use the JNI ICMPv4 interface only, use the following property setting:
+
+### Documentation:
+### http://docs.opennms.org/OpenNMS/${project.version}/documentation/guide-admin/index.html#file-opennms-properties
+
+### General
+
+#org.opennms.dao.ipinterface.findByServiceType=select distinct ipIf from OnmsIpInterface as ipIf join ipIf.monitoredServices as monSvc where monSvc.serviceType.name = ?
```

Comment:
This property needs to be DELETED!!! Why is there HQL in opennms.properties? This must be for some custom behavior somewhere... but whoever needs this should fork their own repo and make the change there.